src/js/angular-tooltips.js

```diff
@@ -77,12 +79,7 @@
 
         thisElement.bind(showTriggers, function onMouseEnterAndMouseOver() {
 
-          if (!initialized) {
-
-            initialized = true;
-            $scope.initTooltip(side);
-          }
-
+          $scope.initTooltip(side);
```

Comment:
Ok not a problem at all, time for a new attribute that solves our troubles:

```js
<a tooltip tooltip-lazy="true/false">
 Lazy re-init or not, just choose
</a>

So that if not in lazy mode you re-init everytime ....